php-checks/src/main/java/org/sonar/php/checks/formatting/IndentationCheck.java

```diff
@@ -54,15 +54,15 @@ private void checkImplementListIndentation(FormattingStandardCheck formattingChe
       int classDecLine = node.getTokenLine();
       int expectedColumn = getLineStartingColumn(node) + PSR2_INDENTATION;
 
-      if (!isOnSameLine(node.getToken(), Iterables.getLast(interfaceList).getToken())) {
+//      if (!isOnSameLine(node.getToken(), Iterables.getLast(interfaceList).getToken())) {
 
         if (!isCorrectlySplittedOnLines(classDecLine, interfaceList)) {
-          formattingCheck.reportIssue("Either split this list into multiple lines or move it on the same line \"" + classDecLine + "\".",
-            interfaceList.get(0));
+//          formattingCheck.reportIssue("Either split this list into multiple lines or move it on the same line \"" + classDecLine + "\".",
+//            interfaceList.get(0));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)